<a href="https://colab.research.google.com/github/munizhernandezalexiscb131-sys/Analisis-de-datos/blob/main/PY_6_Alexis_Mu%C3%B1iz_Hernandez.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [7]:
import pandas as pd
Estudiante = pd.read_csv("datasets/Estuduantes.csv")
Estudiante

NOMBRE  EDAD     GENERO  PROMEDIO
0    Braulio    19  Masculino        77
1     Mihael    19  Masculino        82
2     Aseret    19   Femenino        86
3      Josue    18  Masculino        88
4       Aldo    22  Masculino        80
5       Ivan    19  Masculino        72
6    Vanessa    19   Femenino        73
7   Gabriela    17   Femenino        71
8    Missael    19  Masculino        75
9       Alan    19  Masculino        70
10    Alexis    19  Masculino        90
11    Brayan    19  Masculino        87
12     Diego    19  Masculino        85
13      Luis    20  Masculino        80
14     Efren    20  Masculino        78
15     yamil    22  Masculino        82

In [10]:
Estudiante["EDAD"].describe()

count    16.0000
mean     19.3125
std       1.2500
min      17.0000
25%      19.0000
50%      19.0000
75%      19.2500
max      22.0000
Name: EDAD, dtype: float64

In [11]:
Estudiante["GENERO"].describe()

count            16
unique            2
top       Masculino
freq             13
Name: GENERO, dtype: object

In [13]:
media_edad = Estudiante['EDAD'].mean()
Estudiante["EDAD"].map(lambda edad: edad - media_edad )

0    -0.3125
1    -0.3125
2    -0.3125
3    -1.3125
4     2.6875
5    -0.3125
6    -0.3125
7    -2.3125
8    -0.3125
9    -0.3125
10   -0.3125
11   -0.3125
12   -0.3125
13    0.6875
14    0.6875
15    2.6875
Name: EDAD, dtype: float64

In [14]:
media_edad = Estudiante['EDAD'].mean()
Estudiante["EDAD"].map(lambda edad: edad + media_edad )

0     38.3125
1     38.3125
2     38.3125
3     37.3125
4     41.3125
5     38.3125
6     38.3125
7     36.3125
8     38.3125
9     38.3125
10    38.3125
11    38.3125
12    38.3125
13    39.3125
14    39.3125
15    41.3125
Name: EDAD, dtype: float64

Creamos y cargamos una base de datos de estudiantes con informacion sobre su nombre, edad, género y promedio. A continuación, se efectuan analisis descriptivos de las columnas 'edad' lo cual nos dice que la media es de 19.3125 el minimo es 17 y el maximo 22 y en 'genero' nos dice que hay dos generos y predomina el masculino con 13 datos, y en el primer mapeo de edad observamos que hay pocas personas arriba de la media y en el segundo ahora sumamos la media con la edad